test/data/flask_vulnerable_api/app.py

```diff
@@ -28,6 +29,9 @@ def init_db() -> None:
 
 @app.route("/")
 def index() -> Tuple[Response, int]:
+    filename = request.args.get("filename", "")
```

Comment:
I think it would be cleaner to have a separate test app - I'm afraid this one can grow too big